nuxeo-operation.html

```diff
@@ -210,7 +209,7 @@
 
           // support page provider as input to operations
           // relies on parameters naming convention until provider marshaller is available
-          if (input instanceof Nuxeo.PageProvider) {
+          if (Nuxeo.PageProvider && input instanceof Nuxeo.PageProvider) {
```

Comment:
It may not seem related, but this validation would make our Web UI tests pass!

Reply:
ah! I'll have a look, it probably is related then but for some reason did not come up in the unit tests when landing the change 🤔